yt_dlp/extractor/nytimes.py

```diff
@@ -59,204 +51,546 @@ def get_file_size(file_size):
+            'timestamp': 1429047468,
             'upload_date': '20150414',
             'uploader': 'Matthew Williams',
+            'creator': 'Patricia Cohen',
+            'thumbnail': r're:https?://\w+\.nyt.com/images/.*\.jpg',
+            'duration': 119,
         }
     }, {
-        'url': 'http://www.nytimes.com/2016/10/14/podcasts/revelations-from-the-final-weeks.html',
-        'md5': 'e0d52040cafb07662acf3c9132db3575',
+        # article with audio and no video
+        'url': 'https://www.nytimes.com/2023/09/29/health/mosquitoes-genetic-engineering.html',
+        'md5': '2365b3555c8aa7f4dd34ca735ad02e6a',
         'info_dict': {
-            'id': '100000004709062',
-            'title': 'The Run-Up: ‘He Was Like an Octopus’',
+            'id': '100000009110381',
             'ext': 'mp3',
-            'description': 'md5:fb5c6b93b12efc51649b4847fe066ee4',
-            'series': 'The Run-Up',
-            'episode': '‘He Was Like an Octopus’',
-            'episode_number': 20,
-            'duration': 2130,
+            'title': 'The Gamble: Can Genetically Modified Mosquitoes End Disease?',
+            'description': 'md5:9ff8b47acbaf7f3ca8c732f5c815be2e',
+            'timestamp': 1695960700,
+            'upload_date': '20230929',
+            'creator': 'Stephanie Nolen, Natalija Gormalova',
+            'thumbnail': r're:https?://\w+\.nyt.com/images/.*\.jpg',
+            'duration': 1322,
         }
     }, {
-        'url': 'http://www.nytimes.com/2016/10/16/books/review/inside-the-new-york-times-book-review-the-rise-of-hitler.html',
+        'url': 'https://www.nytimes.com/2023/11/29/business/dealbook/kamala-harris-biden-voters.html',
+        'md5': '3eb5ddb1d6f86254fe4f233826778737',
         'info_dict': {
-            'id': '100000004709479',
-            'title': 'The Rise of Hitler',
-            'ext': 'mp3',
-            'description': 'md5:bce877fd9e3444990cb141875fab0028',
-            'creator': 'Pamela Paul',
-            'duration': 3475,
+            'id': '100000009202270',
+            'ext': 'mp4',
+            'title': 'Kamala Harris Defends Biden Policies, but Says ‘More Work’ Needed to Reach Voters',
+            'description': 'md5:de4212a7e19bb89e4fb14210ca915f1f',
+            'timestamp': 1701290997,
+            'upload_date': '20231129',
+            'uploader': 'By The New York Times',
+            'creator': 'Katie Rogers',
+            'thumbnail': r're:https?://\w+\.nyt.com/images/.*\.jpg',
+            'duration': 97,
         },
         'params': {
+            # inconsistant md5
             'skip_download': True,
         },
     }, {
-        'url': 'http://www.nytimes.com/news/minute/2014/03/17/times-minute-whats-next-in-crimea/?_php=true&_type=blogs&_php=true&_type=blogs&_r=1',
```

Comment:
This URL still works too, although it redirects to `https://archive.nytimes.com/www.nytimes.com/news/minute/2014/03/17/times-minute-whats-next-in-crimea/?_php=true&_php=true&_r=1&_type=blogs&_type=blogs`

We'll probably need to add a new `NYTimesArchive` extractor for this, which can be a separate PR. Just noting it here so that it's not forgotten

Reply:
I'll take this into consideration in the next PR.